catalog.html

```diff
@@ -0,0 +1,316 @@
+            <b class="device-card-title">Любительская
+              селфи-палка</b>
+          </a>
+          <h3 class="device-card-price">500₽</h3>
+        </li>
+
+        <li class="device-card">
+          <a class="device-card-link" href="#">
+            <img class="device-card-image" src="" alt="">
+            <b class="device-card-title">Профессиональная селфи-палка </b>
+          </a>
+          <h3 class="device-card-price">1500₽</h3>
+        </li>
+
+        <li class="device-card">
+          <a class="device-card-link" href="#">
+            <img class="device-card-image" src="" alt="">
+            <b class="device-card-title">Непотопляемая
+              селфи-палка </b>
+          </a>
+          <h3 class="device-card-price">2500₽</h3>
+        </li>
+
+        <li class="device-card">
+          <a class="device-card-link" href="#">
+            <img class="device-card-image" src="" alt="">
+            <b class="device-card-title">Селфи-палка «Следуй за мной»</b>
+          </a>
+          <h3 class="device-card-price">1500₽</h3>
+        </li>
+      </ul>
+      <button class="button pagination-show-more" type="button">Загрузить еще</button>
+
+
+      <ul class="pagination">
+        <li class="pagination-item">
+          <a class="pagination-link pagination-next" href="#">Назад</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">1</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">2</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">3</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">4</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">5</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">6</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">7</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">8</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">9</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">10</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">11</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">12</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">13</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link">14</a>
+        </li>
+
+        <li class="pagination-item">
+          <a class="pagination-link">15</a>
+        </li>
+
+        <li class="pagination-item">
+        </li>
+      </ul>
+      <a class="pagination-link pagination-next" href="#">Вперед</a>
```

Comment:
Думаю, что должно быть так 😏
```suggestion
        <li class="pagination-item">
          <a class="pagination-link pagination-next" href="#">Вперед</a>
        </li>
      </ul>
```